docs/docs/base.md

```diff
@@ -22,6 +22,10 @@ deploy the containers in a K8s cluster and your company has some security restri
 Feel free to modify this image to your specific needs. You are able to set proxies,
 custom registry URLs, your timezone, CA certificates and any other stuff.
 
+The following environment variable can be set in all images:
+
+- `WORKSPACE_DIR`: The directory applications (Eclipse, Capella, Jupyter) will use as workspace.
```

Comment:
Can we add the limitation that only paths below `/workspace` and `/home/techuser` should be used?